packages/create-next-app/create-app.ts

```diff
@@ -62,8 +62,9 @@ export async function createApp({
 
     try {
       repoUrl = new URL(example)
-    } catch (error: any) {
-      if (error.code !== 'ERR_INVALID_URL') {
+    } catch (error: unknown) {
+      const err = error as Error & { code: string | undefined }
```

Comment:
Not sure that casting is really any safer. 🤷  I suppose it's slightly closer to the truth this way.